javascript/javascript1/week1/housepricecalculator.js

```diff
@@ -0,0 +1,22 @@
+const gardenSizeInM2 = [100, 70];
+const width = [8, 5];
+const depth = [10, 11];
+const height = [10, 8];
```

Comment:
I can understand why have you used arrays here, but this makes the code unreadable. This issue in programming called [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)).

To make your code more readable you can create a variable for each person like
```js
const PeterHouseWidth = 8;
const JuliaHouseWidth = 8;
```
or just one variable  `width` and add a comment above it 
```js
// Peter
let width = 8;
...
// Julia
let width = 5;
...
```
Or later you will learn some better ways using objects and classes. 
- It is ok for now, you can leave it. Just wanted to draw your attention to this point.

Reply:
Thankyou for the feedback. should i merge pull request.